dream_layer_backend/comfy_nodes/api_nodes/runway_text2img.py

```
@@ -0,0 +1,326 @@
+                if image.mode != "RGB":
+                    image = image.convert("RGB")
+                
+                # Convert to numpy array and then to torch tensor
+                image_array = np.array(image, dtype=np.float32) / 255.0
+                image_tensor = torch.from_numpy(image_array).unsqueeze(0)  # Add batch dimension
+                
+                print(f"[RunwayText2Img] Image generated successfully: {image_tensor.shape}")
+                return (image_tensor,)
+                
+        except httpx.TimeoutException:
+            raise RuntimeError(f"Request timed out. Try increasing the timeout parameter.")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)